src/components/form/FormFieldSelectTable.vue

```diff
@@ -348,6 +348,19 @@ export default defineComponent({
         </dialog-default>
       </q-card>
     </q-field>
+    <div
+      v-if="variant === 'company'"
+      class="text-caption text-grey-7 q-mt-sm"
+      data-cy="form-select-table-user-note"
+    >
+      <span
+        v-html="
+          $t('form.company.textUserExperience', {
+            email: 'kontakt@dopracenakole.cz',
```

Comment:
E -mail should be obtained from a global config variable `contactEmail` https://github.com/auto-mat/ride-to-work-by-bike-frontend/blob/595c15d657ed3135fe15b5e422e85a0ce4f4fde9/ride_to_work_by_bike_config.toml#L20.

Reply:
Yes, added contact address from config.